src/segger/data/constants.py

```diff
@@ -85,3 +85,31 @@ class MerscopeKeys(Enum):
     COUNTS_CELL_KEY = "cell"
     CELL_X = "center_x"
     CELL_Y = "center_y"
+
+
+class SpatialDataKeys(Enum):
+    """Keys for *MERSCOPE* data (Vizgen platform)."""
+
+    # File mappings
+    TRANSCRIPTS_FILE = "detected_transcripts.csv"
+    BOUNDARIES_FILE = "cell_boundaries.parquet"
+    CELL_METADATA_FILE = "cell_metadata.csv"
+
+    # Cell identifiers
+    CELL_ID = "cell_id"
+    TRANSCRIPTS_ID = "transcript_id"
+
+    # Coordinates and locations
+    TRANSCRIPTS_X = "x"
+    TRANSCRIPTS_Y = "y"
+    BOUNDARIES_VERTEX_X = "center_x"
+    BOUNDARIES_VERTEX_Y = "center_y"
+
+    # Metadata
+    FEATURE_NAME = "<placeholder>"
```

Comment:
Some of these constants should be populated real time by using `spatialdata` APIs/looking at the metadata of the Zarr store. For the moment I inherit from Xenium, and manually ask the user to specify the `FEATURE_NAME`.

Reply:
Exactly. I am already throwing a message to the user, but directly an exception instead of a warning since if the user doesn't pass it's very very likely being wrong.